Python/codegen.c

```diff
@@ -772,6 +772,10 @@ _PyCodegen_Body(compiler *c, location loc, asdl_stmt_seq *stmts, bool is_interac
             assert(st->kind == Expr_kind);
             location loc = LOC(st->v.Expr.value);
             ADDOP_LOAD_CONST(c, loc, cleandoc);
+
+            PySTEntryObject *ste = SYMTABLE_ENTRY(c);
+            ste->ste_has_docstring = 1;
```

Comment:
codegen.c shouldn't be mutating the symtable. This should instead be computed in symtable.c.